File: CS201/Co-Pilot/Starter.py
'''
Determine how many unique words in an array of words start with a specified letter. Write a method that determines how many different strings in words have a first letter equal to first.
'''

def begins(words, first):
    count = 0
    for i in range(len(words)):
        if words[i][0] == first:
            count += 1
    return count

File: CS201/Co-Pilot/Starter.java
/*
Problem Statement:
Determine how many unique words in an array of words start with a specified letter. 
Write a method that determines how many different strings in words have a first letter equal to first.

Constraints:
There will be at most 50 elements in words.
first will be a single character string.
*/

import java.util.*;

public class Starter {
    public int begins(String[] words, String first) {
        int count = 0;
        for (int i = 0; i < words.length; i++) {
            if (words[i].startsWith(first)) {
                count++;
            }
        }
        return count;
    }
}